crates/chia-consensus/src/spendbundle_conditions.rs

```diff
@@ -30,12 +31,18 @@ pub fn get_conditions_from_spendbundle(
     let mut ret = SpendBundleConditions::default();
     let mut state = ParseState::default();
 
-    for coin_spend in &spend_bundle.coin_spends {
-        let byte_cost = (coin_spend.puzzle_reveal.len() + coin_spend.solution.len()) as u64
-            * constants.cost_per_byte;
-
-        subtract_cost(a, &mut cost_left, byte_cost)?;
+    let spends_info = spend_bundle.coin_spends.iter().map(|coin_spend| {
+        (
+            coin_spend.coin,
+            &coin_spend.puzzle_reveal,
+            &coin_spend.solution,
+        )
+    });
+    let spend_generator_length = solution_generator(spends_info)?.len();
+    let byte_cost = spend_generator_length as u64 * constants.cost_per_byte;
```

Comment:
I think the long term goal should be to not actually generate the block-generator, but just compute the byte-cost of it. For now, I think it's fine, to get everything to work first.

You should deduct the 2 bytes for the quote as well

Reply:
Done.